src/utils/DateFormatter.ts

```diff
@@ -0,0 +1,12 @@
+export const dateToString = (date: Date): string => {
```

Comment:
The whole point of having that function is fixing issue around timezone differences although it does not say that either in function name or in JS doc. Make it more obvious that formatted date is in local time.